### ISSUE TITLE
Unable to Use IOS Voiceover with Likes

### ISSUE THREAD

**Issue by kerrynicl** [Automattic/wp-calypso#85302]
### Impacted plugin

Jetpack, Social, None / Other

### Quick summary

From user complaint - 
Recent changes to the presentation of likes have caused VO to not work. The change is a move from larger square gravatars to small round ones and moved the number of likes to the end of the gravatar row. If you click on this, it opens a dropdown list of all likes to the post.  This dropdown list is not the findable by VO it requires you to close VO, select the list visually and then reactivate VO in order read the list. This cannot be done without vision. Previously VO would simply read continuously from the Like Button through the full list of gravatars.

### Steps to reproduce

Turn on IOS Voiceover, click on likes on a post

### A clear and concise description of what you expected to happen.

 Previously VO would simply read continuously from the Like Button through the full list of gravatars.

### What actually happened

it opens a dropdown list of all likes to the post.  This dropdown list is not the findable by VO it requires you to close VO, select the list visually and then reactivate VO in order read the list. 

### Impact

One

### Available workarounds?

No and the platform is unusable

### Logs or notes

User complaint - ZD-7395200

**Comment by ivan-ottinger** [comment]
Thank you for the report of the issue. It has been addressed in D133146-code and D132122-code, and is already fixed in production. Feel free to reopen in case you encounter any related issues.
